Samples/spa-azurefunction/README.md

```diff
@@ -23,6 +23,9 @@ Before proceeding you will need:
 ## App Overview
 This sample app is built in Javascript using React.js for the UI and Azure Functions as the back-end for hosting endpoints to SharePoint Embedded APIs. The UI portion as well as the file management logic (utilizing Microsoft Graph APIs) of the app lives in `/packages/client-app` and the Azure Functions live in `/packages/azure-functions`.
 
+> [!IMPORTANT]
```

Comment:
Can you add this under step 2? Where it says "Install the Azure Functions Extension"

Reply:
I added it, closing this PR as I took your changes directly. Thanks Javier!